cba_settings_userconfig/cba_settings.sqf

```diff
@@ -150,14 +156,14 @@ force force ace_maptools_rotateModifierKey = 1;
 // ACE Medical
 force force ace_medical_ai_enabledFor = 2;
 force force ace_medical_AIDamageThreshold = 1;
-force force ace_medical_allowDeadBodyMovement = true;
 force force ace_medical_allowLitterCreation = true;
 force force ace_medical_allowUnconsciousAnimationOnTreatment = true;
 force force ace_medical_amountOfReviveLives = -1;
 force force ace_medical_bleedingCoefficient = 1;
 force force ace_medical_blood_enabledFor = 2;
 force force ace_medical_consumeItem_PAK = 0;
 force force ace_medical_consumeItem_SurgicalKit = 0;
+force ace_medical_convertItems = 0;

```

Comment:
Why is this disabled?
I am asking because I can't really find any source of what it is doing. But just from reading it, it seems to convert non-ACE medical supplies to ACE ones, which would be nice.

Reply:
It's a fairly new feature of ACE3. I don't know how well it works nor have we tested it. That's why I left it on the default setting which is "off".